sfputil/main.py

```diff
@@ -912,4 +1223,4 @@ def version():
 
 
 if __name__ == '__main__':
-    cli()
+    cli()
```

Comment:
>cli [](http://example.com/codeflow?start=4&length=3)

Is it possible to unit test the new features?

Reply:
added new test cases in sfputil_test.py